apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/page/content/staticelement/request/PageStaticRequests.java

```diff
@@ -0,0 +1,29 @@
+package gov.cdc.nbs.questionbank.page.content.staticelement.request;
+
+import org.springframework.lang.Nullable;
+
+public interface PageStaticRequests {
```

Comment:
issue: There is nothing implementing this interface,it could be sealed having each record implement the interface.

suggestion:  The interface name can help reduce some repetition and express the intention a little more.  Something like, `StaticContentRequests.AddDefault`, `StaticContentRequests.AddHyperlink` like the `PageContentCommand`s.